cypress/integration/pages/testsForAllPages.js

```diff
@@ -188,6 +188,10 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
                     description = body.promo.summary;
                     title = body.promo.headlines.headline;
                     break;
+                  case 'photoGalleryPage':
```

Comment:
This doesn't need to be tackled here, but (for visibility) I've raised an issue here https://github.com/bbc/simorgh/issues/5280 to refactor this switch to make use of our page type constants.